apiml-security-common/src/main/java/org/zowe/apiml/security/common/error/AuthExceptionHandler.java

```diff
@@ -31,150 +33,208 @@
 import org.zowe.apiml.security.common.token.TokenNotProvidedException;
 import org.zowe.apiml.security.common.token.TokenNotValidException;
 
+import java.util.Map;
+import java.util.function.BiConsumer;
+
+import static java.util.Map.entry;
+
 /**
  * Exception handler deals with exceptions (methods listed below) that are thrown during the authentication process
  */
 @Slf4j
 @Component
 public class AuthExceptionHandler extends AbstractExceptionHandler {
 
+    private ApplicationInfo applicationInfo;
+
     @SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
     public AuthExceptionHandler(
         MessageService messageService,
-        ObjectMapper objectMapper) {
-        super(messageService, objectMapper);
+        ObjectMapper objectMapper,
+        @Autowired(required = false) ApplicationInfo applicationInfo) {
```

Comment:
Is it still valid that applicationInfo could be null?

Reply:
@taban03 can you take this one?

Reply:
yes

Reply:
Yes, it's still valid: i.e. when running the catalog, spring won't find the bean and will fail

Reply:
I am not sure how important it is, but for sure I have created the issue #4183